### DISCUSSION TITLE
Language Support 🌎

### DISCUSSION THREAD

**Discussion by Martinxux** [Daniele-rolli/Beaver-Notes#25] [Ideas]
we need more language support

**Comment by Daniele-rolli** [comment]
Thanks for the feedback! Do you mean language support as in spell-check or at a UI level?

**Comment by Martinxux** [comment]
Sorry for not making it clear, I mean UI level language support

**Comment by Daniele-rolli** [comment]
It’s going to be on top of my priority list once the mobile app is out, but it’s going to take a community effort to translate the UI into multiple languages.

**Comment by Daniele-rolli** [comment]
Hi everyone, I've turned this issue into a discussion since the next version of Beaver will introduce support for multiple languages. If you are willing to help translate Beaver Notes into your own language, please stay tuned. As a new page in the documentation, along with an article, is set to be released on October 31st. This time, I'm making the code available on GitHub, however, I'll wait a bit before packaging the new release so that everyone has the opportunity to add their own language to the code.

**Comment by Daniele-rolli** [comment]
Oh my, this discussion has spiderwebs, judging by the activity in here. Well anyway, I am proud to say we've made huge steps forward in terms of language support with the introduction of Simplified Chinese, German, and Italian. Other languages, such as Dutch, are also on the way.
